west.yml

```diff
@@ -135,7 +135,7 @@ manifest:
       revision: c39888ff74acf421eeff9a7514fa9b172c3373f7
     - name: trusted-firmware-m
       path: modules/tee/tfm
-      revision: 58c70aeb9937c3b6b97a07cc654e48b2c56c258a
+      revision: pull/10/head
```

Comment:
Is this ref available, without adding an explicit fetch for prs?

Reply:
Guess so